dags/longitudinal.py

```diff
@@ -111,9 +111,25 @@
     output_visibility="private",
     dag=dag)
 
+taar_lite_guidranking = EMRSparkOperator(
+    task_id="taar_lite_guidranking",
+    job_name="TAARlite Addon Ranking",
+    owner="mlopatka@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com"],
+    execution_timeout=timedelta(hours=2),
+    instance_count=4,
+    env=mozetl_envvar("taar_lite_guidranking",
+                      {"date": "{{ ds_nodash }}"},
+                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
+    release_label="emr-5.8.0",
```

Comment:
The current release version of emr is 5.9.0. You may not need to specify this anymore.